mailer.rst

```diff
@@ -420,6 +420,29 @@ setting the ``auto_tls`` option to ``false`` in the DSN::
 
     This setting only works when the ``smtp://`` protocol is used.
 
+Require use of TLS
```

Comment:
Maybe:

```suggestion
Enforce TLS
```

Reply:
Happy to change if preferred, but `Enforce` feels slightly less accurate to me. `Enforce` sounds like it'd change/force the connection in some manner and/or force TLS upon the whole connection process, whereas this option doesn't change the connection in any way, just ensures it's eventually used.

Reply:
I understand your concern about the term "Enforce" and I agree with you.
What do you think about "**Ensure TLS**" instead? It might better reflect that the option guarantees the use of TLS without actively altering the connection.

Reply:
@Spomky Sorry for the late response. I agree that "Ensure TLS" would be better. I've updated the heading to that.